packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -1,30 +1,36 @@
+import isNil from 'lodash.isnil'
 /**
  * @function copyEmptyArrayProps
  * @description Takes two objects and if there are Array props in oldObj which doesn't exist in newObj, then copy it with an empty value.
  * @param {Object} oldObj
  * @param {Object} newObj
  * @returns {Array} Ordered Array [oldObj, newObj]
  */
-export default function copyEmptyArrayProps(oldObj, newObj) {
-  const newObjWithFixedEmptyArray = Object.entries(oldObj).reduce(
-    (acc, [key, value]) => {
+export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
```

Comment:
if we intend to spread `newObj`, we need to fallback to an `Object` if the passed value is undefined

Reply:
Technically it actually works to spread `undefined`, but I agree that this makes it easier to follow.

```
const a = {...undefined};
console.log(a);
// {}
let b;
const c = {...b}
console.log(c);
// {}
```